src/components/EventModal/EventOwner.tsx

```diff
@@ -63,6 +73,45 @@ export default function EventOwner(props: EventOwnerProps) {
     );
   }
 
+  const hideShowScanner = () => {
```

Comment:
is this meant to toggle? if it is you can just do `setScannerOpen(!scannerOpen)`